### DISCUSSION TITLE
Automate Windows server/client certificate

### DISCUSSION THREAD

**Discussion by yanbiabeu** [smallstep/certificates#380] [General] [answered]
Hello team,
I would like to automate the creation of Windows client/server certificates, this will allow all Windows ec2 instances to have a certificate generated from our certificate manager in AWS. the certificate manager I'm using is a sub-CA of the root. The environment is managed by Puppet.
I'm not very familiar with configuring the CA certificate, I'm wondering if anyone could share their personal experience with this project and some guides.

**Comment by dopey** [comment, marked as answer]
Hey,

I replied in Gitter, but I'll copy the response here as well for posterity.

> Hey, it sounds like you’d like to create an intermediate CA using an existing root. We have documentation for that scenario here: https://github.com/smallstep/certificates/blob/master/docs/questions.md#i-already-have-pki-in-place-can-i-use-this-with-my-own-root-certificate.

Take a look at that documentation and if you have any questions about how to set that up we can definitely help you out.
